src/Sql/dbo/Stored Procedures/OrganizationIntegrationConfiguration_ReadByOrganizationIntegrationId.sql

```diff
@@ -0,0 +1,13 @@
+CREATE PROCEDURE [dbo].[OrganizationIntegrationConfiguration_ReadByOrganizationIntegrationId]
```

Comment:
```suggestion
CREATE PROCEDURE [dbo].[OrganizationIntegrationConfiguration_ReadManyByOrganizationIntegrationId]
```
Will have to fix this in a few places.